data/seeds/002_Avatars.js

```diff
@@ -1,14 +1,68 @@
 exports.seed = function (knex) {
   return knex('Avatars').insert([
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-1.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-2.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-3.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-4.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-5.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-6.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-7.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-8.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-9.svg' },
-    { AvatarURL: 'https://labs28-b-storysquad.s3.amazonaws.com/hero-10.svg' },
+    { AvatarURL: 'https://storysquad-main.s3.amazonaws.com/avatars/01.svg' },
```

Comment:
looks like we need to change the name of the s3 bucket, or make a new one for scribble stadium.

Reply:
good idea! I created a new s3 bucket and updated those seed files.